tests/api/test_initial_data.py

```diff
@@ -200,6 +208,24 @@ def test_add_default_hub_source_if_needed():
         assert update_default_hub_source.call_count == 0
 
 
+def test_migrate_project_summaries():
+    db, db_session = _initialize_db_without_migrations()
+
+    # Create a project
+    project = mlrun.common.schemas.Project(
+        metadata=mlrun.common.schemas.ProjectMetadata(name="project-name"),
+    )
+    db.create_project(db_session, project)
```

Comment:
What is the purpose of this test? When you create a project, it also creates the project summary.